src/charts/chart.js

```diff
@@ -474,48 +403,53 @@ export default function Chart (_container) {
     cache.svg.on(".", null).remove()
   }
 
-  function csvToJson (csv) {
-    let json = {"series": []};
+  function getAxisChart () {
+    const { chartType } = config;
+    return (chartType === "line" || chartType === "area" || chartType === "stackedArea") ? Axis(cache.chart) : null;
+  }
+
+  function getLineChart () {
+    const { chartType } = config;
+    return (chartType === "line" || chartType === "area" || chartType === "stackedArea") ? Line(cache.panel) : null;
+  }
 
+  function csvToJson (csv) {
+    //let json = {"series": []};
     let labels = [];
     let addLabel;
 
     csv.forEach((row, index) => {
-      if(index === 0) {
+      if (index === 0) {
         labels.push(getObjectToPush(row[0], index, 0, []));
       } else {
         addLabel = true;
         labels.forEach(label => {
-          if(row[0] === label["label"]) {
+          if (row[0] === label["label"]) {
             addLabel = false;
           }
         });
-
-        if(addLabel) {
-          labels.push(getObjectToPush(row[0], index, 0, []));
-        }
+        if (addLabel) labels.push(getObjectToPush(row[0], index, 0, []));
       }
     });
 
     let __index__;
     labels.forEach((label, index) => {
       __index__ = 0;
       csv.forEach((key, value) => {
-        if(label["label"] === key[0]) {
+        if (label["label"] === key[0]) {
           label["values"].push({
             value: key[1],
             key: __index__
           });
           __index__++;
         }
       })
-    })
+    });
 
-    json["series"] = labels;
-    return json;
+    return { "series": [...labels] };
   }
 
-  function getObjectToPush(label, id, group, values) {
+  function getObjectToPush (label, id, group, values) {
```

Comment:
I see you formatted the code with a space between the function name and (, but the space was removed in lot of other places, not a problem, but probably you can reapply the auto-format used by the IDE, for this file and some others having the same issue.

Reply:
Formatted